pandas/core/construction.py

```diff
@@ -379,6 +381,16 @@ def array(
     ):
         return TimedeltaArray._from_sequence(data, dtype=dtype, copy=copy)
 
+    elif lib.is_np_dtype(dtype, "mM"):
+        warnings.warn(
+            r"dt/td64 dtypes with 'm' and 'h' resolutions are deprecated."
+            r"Supported resolutions are 's', 'ms','us', and 'ns'. "
+            r"In future releases, 'm' and 'h' resolutions will be cast to the closest "
```

Comment:
dont specify "m" and "h", since that leaves out others and it isnt worth listing all of them